### PR TITLE
#483 save signature on cancel

### PR DESCRIPTION
**Pull request by otselnik**

### CODE REVIEW

File: proxy/indexer/indexer.py [hunk 1]
```diff
@@ -586,6 +586,7 @@ def execute(self) -> bool:
             return self._decoding_fail(tx, f'cannot find storage {tx}')
 
         tx.neon_res.clear()
+        tx.neon_res.sol_sign = self.ix.sign.slot
```
Comment (afalaleev) [inline proxy/indexer/indexer.py]: tx.neon_res.sol_sign = self.ix.sign

**Review by afalaleev — CHANGES_REQUESTED** [review]
The additional place IterativeNeonTxSender::_on_success_send in transaction_sender.py

File: proxy/indexer/indexer.py (hunk 1 above)
Comment (otselnik) [reply proxy/indexer/indexer.py]: fixed

File: proxy/indexer/indexer.py [hunk 2]
```diff
@@ -709,9 +708,7 @@ def process_receipts(self):
         # cancel transactions with long inactive time
         for tx in self.state.iter_txs():
             if tx.storage_account and abs(tx.slot - self.current_slot) > CANCEL_TIMEOUT:
-                if not self.unlock_accounts(tx):
-                    tx.neon_res.slot = self.indexed_slot
```
Comment (afalaleev) [inline proxy/indexer/indexer.py]: This was the fix of the 
https://github.com/neonlabsorg/proxy-model.py/issues/477

**Review by afalaleev — APPROVED** [review]

**Pull request merged**